.github/workflows/ci.yml

```diff
@@ -19,28 +19,69 @@ concurrency:
 
 jobs:
   prepare:
+    name: Check how tests should be executed
     runs-on: ubuntu-20.04
     outputs:
       uuid: ${{ steps.uuid.outputs.value }}
       commit-message: ${{ steps.commit-message.outputs.value }}
+      is-label-present: ${{ steps.label-present.outputs.value }}
+      cypress: ${{ steps.cypress.outputs.value }}
+      synpress: ${{ steps.synpress.outputs.value }}
+
     steps:
       - name: ⬇️ ・Checkout repo
         uses: actions/checkout@v3
         with:
           fetch-depth: '50'
 
+      - name: Dump GitHub context
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+        run: |
+          echo "$GITHUB_CONTEXT"
```

Comment:
I think we can remove this.

Reply:
I will leave it for a few runs (mostly when merging to develop) to see if everything works good